README.md

```diff
@@ -104,12 +104,58 @@ If you are using the types in a `commonjs` module, like in a Node app, you just
 ## Trouble shooting and known issues.
 
 ### Web3 and Angular
+
+### New solution
+
+If you are using Angular version >11 and run into an issue building, the old solution below will not work. This is because polyfills are 
+not included in the newest version of Angular.
```

Comment:
```suggestion
If you are using Angular version >11 and run into an issue building, the old solution below will not work. This is because polyfills are not included in the newest version of Angular.
```

Remove new line